topics/transcriptomics/tutorials/scrna-intro/slides_ES.html

```diff
@@ -0,0 +1,1262 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+video: true
+title: "Una introducción al análisis de datos scRNA-seq"
+
+subtopic: single-cell
+priority: 1
+level: Intermediate
+video: true
+zenodo_link: ""
+tags:
+  - single-cell
+questions:
+  - ¿Cómo se comparan las muestras?
+  - ¿Cómo son capturadas las células?
+  - ¿Cuál  es la diferencia entre los métodos Bulk RNA-seq y scRNA-seq?
+  - ¿Por qué es importante clusterizar datos?
+objectives:
+  - Entender las dificultades en la secuenciación y en la amplificación del scRNA-seq y cómo superarlas.
+  - Conocer los tipos de variación en un análisis y cómo controlarlos.
+  - Comprender qué es la reducción de dimensiones y cómo se puede realizar.
+  - Familiarizarse con las principales técnicas de clusterización de datos y cuándo usarlas.
+time_estimation: 30m
+key_points:
+  - Los datos scRNA-seq requieren ser pre-procesados antes que un análisis sea llevado a cabo.
+  - Los grupos de células con perfiles similares son comparados contra otros grupos.
+  - Los problemas de detectabilidad requieren una atención especial en todas las etapas.
+  - La clusterización es una etapa completa del análisis.
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-preprocessing
+lang: es
+translations:
+  - en
+contributors:
+  - mtekman
+  - ales-ibt
+voice:
+  id: Lupe
+  lang: es-US
+  neural: true
+
+---
+
+# Single-cell RNA-seq
+
+Una introducción al análisis de datos scRNA-seq
+
+???
+- Hola a todos y bienvenidos al taller de análisis single cell RNA-seq de Galaxy
+Greetings everybody and welcome to the Galaxy single cell RNA-seq analysis workshop.
+- Aquí les enseñaremos algunos de los conceptos básicos del análisis de datos single cell.
+---
+
+
+## RNA-seq conjunto
+
+.pull-left-large[![](../../images/scrna-intro/rna_cells_bulkrez.svg)]
+
+.pull-right[
+.reduce90[
+.center[*Resumen*]
+
+| | |
+|-:|:-|
+| Resolución| Tejido entero |
+| Señal | Expresión génica promedio por tejido |
+| Expresión diferencial | Diferencia entre la expresión génica promedio entre los tejidos |
+]
+]
+
+
+???
+- Vamos a comenzar con las diferencias entre Bulk RNA-seq y single cell RNA-seq data.
+- Con el método Bulk RNA-seq observamos la expresión promedio de cada gen detectado en cada uno de los tejidos.
+- Debido al número de moléculas de ARN que están siendo consideradas, la profundidad de la secuenciación y la fuerza del análisis son razonablemente altas.
+- La expresión diferencial se mide entonces como la expresión relativa de un gen dado entre un tejido y otro.
+
+---
+
+## Single Cell RNA-Seq
+
+.pull-left-large[![](../../images/scrna-intro/rna_cells_singlerez.svg)]
```

Comment:
The errors are due to missing alt text (image description for e.g. screen readers) on the images, you can add a description of the images inside the square brackets:
 
```suggestion
.pull-left-large[![A description of the image here](../../images/scrna-intro/rna_cells_singlerez.svg)]
```

Reply:
We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

Reply:
We've added in the translations now for the alt-text